AddOnKit/Software/IO Test/deploy.sh

```diff
@@ -0,0 +1,3 @@
+cp io-test.binary run.bin
+cp run.bin /Volumes/Untitled/
```

Comment:
That is a good idea.  Perhaps it could be a command line parameter that if not present is prompted for.  That way you can run the command over and over again without having to answer prompts.